addons/udes_stock/wizard/stock_picking_to_batch.py

```diff
@@ -0,0 +1,32 @@
+from odoo import api, models
+from odoo.exceptions import UserError
+from odoo.tools.translate import _
+
+
+class StockPickingToBatch(models.TransientModel):
+    _inherit = "stock.picking.to.batch"
+
+    def _check_picking_priority(self):
+        """Check picking priority
+
+        Args:
+            pickings (stock.picking): Stock picking object
+
+        Raises:
+            UserError: Raises exception when selected picking has different priority then batch
+        """
+        self.ensure_one()
+        picking_ids = self.env.context.get("active_ids")
+        pickings = self.env["stock.picking"].browse(picking_ids)
```

Comment:
Nit: `self.env["stock.picking"]` should be in a variable at the start of the method, e.g.:
`Picking = self.env["stock.picking"]`

And then:
`pickings = Picking.browse(picking_ids)`

Reply:
done